hedera-node/src/main/java/com/hedera/services/store/models/Token.java

```diff
@@ -110,12 +121,19 @@ public void burn(
 		changeSupply(treasuryRelationship, -serialNumbers.size(), FAIL_INVALID);
 	}
 
+	/**
+	 * Minting unique tokens creates new instances of the given base unique token.
+	 * Increments the serial number of the given base unique token, and assigns each of the numbers to each new unique token instance.
+	 * @param ownershipTracker - a tracker of changes made to the ownership of the tokens
+	 * @param treasuryRel - the relationship between the treasury account and the token
+	 * @param metadata - a list of user-defined metadata, related to the nft instances.
+	 * @param creationTime - the consensus time of the token creation
```

Comment:
nit: It is not the token creation time. Someone might confuse it with the timestamp when the token entity was created. Lets change it to the `Consensus time of the mint transaction`